docs: federation v2.5 documentation updates (#2688)
apollographql/federation: docs/source/federated-types/federated-directives.mdx

</tbody>
</table>

> ## Security

Reviewer: ```suggestion
## Controlling access
```

Reviewer: Making this a gerund to match sibling headers.

### `@authenticated`

> ⚠️ **This directive is available in Apollo Federation 2.5 and later.**

> > ⚠️ **This is an [Enterprise feature](/router/enterprise-features) of the Apollo Router.** It requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/).</br>

Reviewer: ```suggestion
> **It is an [Enterprise feature](/router/enterprise-features) of the Apollo Router** and requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/).</br>
```

Avoiding double ⚠️⚠️

> > ⚠️ **This is an [Enterprise feature](/router/enterprise-features) of the Apollo Router.** It requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/).</br>

Reviewer 2: Something here is [causing a syntax error in the MDX](https://app.netlify.com/sites/apollo-federation-docs/deploys/64b84529825acf0008f80831#L185). I suspect it's the `</br>` tag, but it's just a guess. Maybe try removing them here and elsewhere. We don't have any usages of them in mdx files anywhere.

Author: MDX supports HTML but I think Gatsby was having some issues -> I changed it to match other break tags

>
> If your organization _doesn't_ currently have an Enterprise plan, you can test out this functionality by signing up for a free [Enterprise trial](/graphos/org/plans/#enterprise-trials).

```graphql
directive @authenticated on
    FIELD_DEFINITION
  | OBJECT
  | INTERFACE
  | SCALAR
  | ENUM
```

> Indicates to composition that target element is accessible only to the authenticated supergraph users. For more granular access control see [`@requiresScope`](#requiresScope) directive below. See [JWT Authentication in the Apollo Router](/router/configuration/authn-jwt) for additional details.

Reviewer: ```suggestion
Indicates to composition that the target element is accessible only to the authenticated supergraph users. For more granular access control, see the [`@requiresScope`](#requiresScope) directive below. Refer to the [Apollo Router article](/router/configuration/authorization#authenticated) for additional details.
```

Links to the more detailed WIP page in Router docs. (See https://github.com/apollographql/router/pull/3449/files)

### `@requiresScope`

> ⚠️ **This directive is available in Apollo Federation 2.5 and later.**

> > ⚠️ **This is an [Enterprise feature](/router/enterprise-features) of the Apollo Router.** It requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/).</br>

Reviewer: ```suggestion
> **Is is an [Enterprise feature](/router/enterprise-features) of the Apollo Router** and requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/).</br>
```

>
> If your organization _doesn't_ currently have an Enterprise plan, you can test out this functionality by signing up for a free [Enterprise trial](/graphos/org/plans/#enterprise-trials).

```graphql
directive @requiresScopes(scopes: [federation__Scope!]!) on
    FIELD_DEFINITION
  | OBJECT
  | INTERFACE
  | SCALAR
  | ENUM
```

> Indicates to composition that target element is accessible only to the authenticated supergraph users with appropriate JWT scope. See [JWT Authentication in the Apollo Router](/router/configuration/authn-jwt) for additional details.

Reviewer: ```suggestion
Indicates to composition that the target element is accessible only to the authenticated supergraph users with the appropriate JWT scopes. Refer to the [Apollo Router article](/router/configuration/authorization#requiresscopes) for additional details.
```

#### Arguments

<table class="field-table">
  <thead>
    <tr>
      <th>Name /<br/>Type</th>
      <th>Description</th>
    </tr>
  </thead>

<tbody>

<tr class="required">
<td>

##### `scopes`

`[federation__Scope!]!`

</td>
<td>

> **Required.** List of JWT scopes that have to be granted to the user in order to access the underlying element data.

Reviewer: ```suggestion
**Required.** List of JWT scopes that must be granted to the user in order to access the underlying element data.
```
